src/client/js/main.js

```diff
@@ -0,0 +1,9 @@
+require.config({
+    baseUrl: 'js',
```

Comment:
instead of having ../../../ on all our components, can we extend the baseUrl to have /js/<dir>/<dir> etc... ?